R/utils.R

```diff
@@ -169,6 +169,9 @@ try_silently <- function(expr) {
 viapply <- function(x, ...) vapply(x, ..., FUN.VALUE = integer(1))
 vcapply <- function(x, ...) vapply(x, ..., FUN.VALUE = character(1))
 
+# imitate sQuote(x, q) [requires R>=3.6]
+quote_wrap <- function(x, q) sprintf("%1$s%2$s%1$s", q, x, q)
```

Comment:
The third argument to sprintf is unused, but I think `paste0(q, x, q)` would be more readable anyway.

Reply:
Agreed. I saw this function in the testthat helpers too, I'll just pull that into utils.